core/code/map_data_request.js

```diff
@@ -74,7 +74,7 @@ window.MapDataRequest = function() {
   var _this = this;
   addHook('portalDetailLoaded', function(data){
     if(data.success) {
-      _this.render.processGameEntities([data.ent]);
+      _this.render.processGameEntities([data.ent], 'detailed');
```

Comment:
should we call `createPortalEntity(data.ent, 'detailed)` instead

Reply:
```suggestion
      _this.render.createPortalEntity(data.ent, 'detailed');
```

Reply:
Good point!